index.js

```diff
@@ -1,15 +1,15 @@
-const Hoek = require('hoek');
 const server = require('./server.js');
 const plugins = require('./server-plugins.js');
 const routes = require('./routes/routes.js')
 
-server.register(plugins, err => {
-  Hoek.assert(!err, err);
+const start = async function() {
+
+  await server.register(plugins);
 
   server.route(routes);
 
-  server.start(err => {
-    Hoek.assert(!err, err);
-    console.log('Server running at: ' + server.info.uri);
-  })
-});
\ No newline at end of file
+  await server.start();
+  console.log('Server running at: ' + server.info.uri);
+}
+
+start();
```

Comment:
Can you add the graceful stop like here while at it: https://github.com/nzzdev/Q-server-nzz/blob/dev/index.js#L170-L184

Reply:
Ah sure, I'll add that for the other PRs as well